internal/controller/postgrescluster/controller.go

```diff
@@ -136,6 +136,14 @@ func (r *Reconciler) Reconcile(
 			return runtime.ErrorWithBackoff(err)
 		}
 	}
+	// Issue Warning Event if postgres version is EOL according to PostgreSQL:
+	// https://www.postgresql.org/support/versioning/
+	currentTime := time.Now()
+	if postgres.ReleaseIsFinal(cluster.Spec.PostgresVersion, currentTime) {
+		r.Recorder.Eventf(cluster, corev1.EventTypeWarning, "EndOfLifePostgresVersion",
+			"Postgres Version %v is End-Of-Life. We recommend upgrading to the latest supported version.",
```

Comment:
Perhaps something like

"Postgres version %v is the last release of major version %m. We recommend upgrading... blah blah"

Reply:
Hmm... I think we would have to exec into the postgres container to get the minor version. The `cluster.Spec.PostgresVersion` is just the major version.

Reply:
Hmmm...

Is %v the major and minor or just the major?

"Postgres version %v is the last release of this version. We recommend upgrading... blah blah" may work.

Reply:
Also, we won't necessary know that they are using the last release of a particular major version. We just know that the Major Version they are using is End-Of-Life.

Reply:
Ok good context thanks. 


"Postgres version %v is nearing the end of community support. We recommend upgrading. See https://www.postgresql.org/support/versioning/."